implementations/rust/ockam/ockam_api/src/nodes/models/portal.rs

```diff
@@ -150,11 +150,16 @@ impl CreateOutlet {
 #[rustfmt::skip]
 #[cbor(map)]
 pub struct InletStatus {
+	#[serde(rename = "listen_address")]
     #[n(1)] pub bind_addr: String,
+    #[serde(skip)]
     #[n(2)] pub worker_addr: String,
+    #[serde(skip)]
     #[n(3)] pub alias: String,
     /// An optional status payload
+    #[serde(skip)]
     #[n(4)] pub payload: Option<String>,
+	#[serde(rename = "route_to_outlet", serialize_with = "super::address::serialize_route_as_multiaddr")]
     #[n(5)] pub outlet_route: String,
```

Comment:
There is a major problem with this approach here: the skip/rename rules you are adding here are tailored to match one specific use case: printing this struct in the CLI. So, instead of doing that, I'd propose to create a new struct in the `ockam_command` crate with just the fields that you need.

The same goes for the `OutletStatus` struct.

Reply:
Fixed